core/src/integrationTest/java/org/lflang/tests/runtime/PythonTest.java

```diff
@@ -70,7 +70,7 @@ protected boolean supportsSingleThreadedExecution() {
 
   @Override
   protected boolean supportsDockerOption() {
-    return false; // FIXME: https://issues.lf-lang.org/1564
```

Comment:
While it's disappointing that it took so long for this to finally to get fixed, I'm glad there was a reference to the issue in the source code, so at least that mechanism is working as it's supposed to.